source/javascripts/components/GuidedOnboarding/GuidedOnboardingContent/Steps.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Flex, Link, Text } from "@bitrise/bitkit";
+
+import "./GuidedOnboardingContent.scss";
+
+import stepStatusNextIcon from "../../../../images/step-status-next-icon.svg";
+import statusSuccessfulIcon from "../../../../images/status-successful-icon.svg";
+
+const appSteps = [
+    {
+        title: "add new app",
+        isSuccessful: true
+    },
+    {
+        title: "run a build",
+        isSuccessful: true
+    },
+    {
+        title: "configure your workflows",
+        isSuccessful: false
+    }
+];
+
+export const Steps: React.FC = () => (
+    <Flex
+        className="guided-onboarding-step-row"
+        direction="horizontal"
+        alignChildrenVertical="middle"
+        alignChildrenHorizontal="between">
+        <Flex direction='horizontal' alignChildrenVertical='middle' gap='x4'>
+        {
+            appSteps.map(({title, isSuccessful}) => (
+                <Flex key={title} direction='horizontal' gap='x2' alignChildrenVertical='middle'>
+                    <img src={isSuccessful ? statusSuccessfulIcon : stepStatusNextIcon} />
+                    <Text size='2' uppercase={true} style={{ lineHeight: "16px" }}>{title}</Text>
```

Comment:
If it is always true I would just use the defaulted version
```suggestion
                    <Text size='2' uppercase style={{ lineHeight: "16px" }}>{title}</Text>
```